modules/middleware/src/buildQuery/index.js

```diff
@@ -62,7 +63,7 @@ function getRegexFilter({ nestedFields, filter }) {
   const esFilter = wrapFilter({
     filter,
     nestedFields,
-    esFilter: { regexp: { [field]: value.replace('*', '.*') } },
+    esFilter: { regexp: { [field]: value.replace(REGEX, '') } },
```

Comment:
This is certainly better than the old implementation, but will definitely be breaking for any applications using it as it was. If you're not in need of this change, perhaps we should remove the change for now and open an issue to fix this properly with the next major version.

Reply:
that works for me. No we dont seem to be using it.